angular-workspace/projects/ni/nimble-angular/src/testing/async-helpers.ts

```diff
@@ -1,3 +1,5 @@
+// This is the one production file that is allowed to access testing code.
```

Comment:
It's not "allowed", it's a bug until we create a `testing` secondary entrypoint.

Update the description to emphasize it is a workaround and link to the following: `https://github.com/ni/nimble/issues/172#issuecomment-1529958809`